test/extensions/filters/network/thrift_proxy/conn_manager_test.cc

```diff
@@ -509,6 +509,16 @@ TEST_F(ThriftConnectionManagerTest, OnDataHandlesTransportApplicationException)
   EXPECT_EQ(0U, stats_.request_active_.value());
 }
 
+TEST_F(ThriftConnectionManagerTest, OnDataHandlesGarbageRequest) {
```

Comment:
Can you add `//` comments above each `TEST_F` explaining that it's a regression for the specific bug.